kube/minikube/scripts/deploy.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Function to check if a resource exists
+check_resource_exist() {
+  local kind=$1
+  local name=$2
+
+  # Wait for the resource to be available
+  while [[ -z "$(kubectl get $kind $name -o json)" ]]; do
```

Comment:
How about `kubectl wait`?

Reply:
Yes, it looks like a better option. I'll update the script.
Thanks for pointing it out.

Reply:
It turns out that `kubectl wait` is not useful for checking resource existence.
Please see https://github.com/kubernetes/kubernetes/issues/83242
Also, the `wait` tool is experimental. Hence we should keep `kubectl get` instead.

Reply:
Thanks for sharing the link.
I have added `kubectl wait` for deployments.